package.json

```diff
@@ -62,6 +58,7 @@
     "chai": "3.5.0",
     "conventional-changelog": "^1.1.0",
     "css-loader": "0.26.1",
+    "del": "^2.2.2",
```

Comment:
Actually I don't like to use fixed numbers because you will use outdated versions for long time in most cases. As well it appears really rarely that a new version breaks something.
But I understand that it as well makes sense and therefore I'll change it :)